File: Assignment_No-12/Question_02.c
/*
2)write a program which accept number from user and return the count of odd digits.
input=2395
output:3
input=-1018
output=2
*/
#include<stdio.h>
int OddCount(int iNo)
{
	int iCnt=0;
	int iCount=0;
	if(iNo<0)
	{
		iNo=-iNo;
	}
	while(iNo>0)
	{
		iCnt=iNo%10;
		if(iCnt%2!=0)
		{
			iCount++;
		}
		iNo=iNo/10;
	}
	return iCount;
}
int main()
{
	int iValue=0;
	int iRet=0;

	printf("Enter a number:");
	scanf("%d",&iValue);

	iRet=OddCount(iValue);

	printf("%d",iRet);
	return 0;
}

File: Assignment_No-36/Question_02.java
/*2)write a java program which accept number from user and return the count odd digit

input:2395

output:3

*/

////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Digits
{
    public void CountoddDigits(int iNo)
    {
        int iCount = 0;
        int iDigit = 0;

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            if(iDigit%2!=0)
            {
                iCount++;
            }
            iNo = iNo / 10;
        }
        System.out.println("Frequency of odd  is : "+iCount);
    }
}

class program186
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        
        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Digits dobj = new Digits();
        dobj.CountoddDigits(iValue);

    }
}
